app/helpers/reconciliations_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ReconciliationsHelper
+  def reconciliation_delta_table_row(delta)
+    link_to_sheet = inventory_reconciliation_count_sheet_path(delta.reconciliation, delta.warning_count_sheet_id)
+    href_data = delta.warning_count_sheet_id ? { href: link_to_sheet } : {}
+    data = { toggle: "tooltip", placement: "top", title: delta.warning_text }.merge(href_data)
+    tag(:tr, class: delta.row_css_class, data: data)
```

Comment:
So we are not having floating ending tags in our html please implement a block format. Additionally the reformat to remove the ternary was requested by @smellsblue as he abhors them.

```
    data = { toggle: "tooltip", placement: "top", title: delta.warning_text }.merge(href_data)

    if delta.warning_count_sheet_id
      data[:href] = inventory_reconciliation_count_sheet_path(delta.reconciliation, delta.warning_count_sheet_id)
    end

    tag(:tr, class: delta.row_css_class, data: data) do
      yield
    end
```

Reply:
I made an attempt at this and couldn't get the rows to populate with the yield.  Feel free to commit to this PR and fix it, I spent too much time trying to figure it out!